### ISSUE TITLE
Multiple Upload endpoint 

### ISSUE THREAD

**Issue by brishtiteveja** [dtube/DTubeGo#249]
Currently we already post a content to Hive. But we do it after the uploading process is completed. In the case of 3rd party upload, We can possibly eliminate the manual (user himself uploads) upload by connecting user's social account with dtubego account.

Therefore, we can take the burden of uploading (at least for shorts) away from users and also provide them an easy way (A feature) to publish videos to multiple social media platform all at once. May be we can add dtube links too.. so that their followers jump back to the original link on dtube.

I only thought about Youtube API before. But I also figured that Imgur also allows video upload. Check below.

Requirements: This requires authorization/authentication of ImGur(imgur.com) or Youtube(youtube.com). Through profile page, make user add his account/channel with Imgur Account API (https://apidocs.imgur.com/#a94d108b-d6e3-4e68-9521-47ea79501c85) and Youtube DATA API (https://developers.google.com/youtube/v3/getting-started#before-you-start)

Imgur Upload API:https://apidocs.imgur.com/#c85c9dfc-7487-4de2-9ecd-66f727cf3139
Youtube Video Upload API: https://developers.google.com/youtube/v3/guides/uploading_a_video

**Comment by tibfox** [comment]
good idea but I think it could be very hard to implement this. We will see when the time is right and a future proposal got approved. I think we will need several days to code and test this. And also there are probalby many restrictions we havew to research before implmenetation. The IG API for example does not support creator-accounts
